src/droidy/droidy-host-session.vala

```diff
@@ -210,6 +210,15 @@ namespace Frida {
 			);
 		}
 
+		public async HashTable<string, Variant>? query_system_parameters (Cancellable? cancellable) throws Error, IOError {
+			try {
+				var server = yield get_remote_server (cancellable);
+				return yield server.session.query_system_parameters (cancellable);
```

Comment:
We will also need a fallback here for non-rooted Android (and similarly in the Fruity backend for jailed iOS).